packages/frontend/src/common/api.js

```diff
@@ -0,0 +1,47 @@
+import Vue from 'vue';
+import axios from 'axios';
+import VueAxios from 'vue-axios';
+import AxiosPlugin from 'vue-axios-cors';
+
+const ApiService = {
+  async init() {
+    Vue.use(VueAxios, axios);
+    Vue.use(AxiosPlugin);
+    Vue.axios.defaults.baseURL = 'http://localhost:8001/';
```

Comment:
How do you dynamically set Vue.axios.defaults.baseURL? The environment variables do not get passed to the client machine.

Reply:
something about Vuex.store... need more research